splitgraph/sgfile/execution.py

```diff
@@ -243,6 +249,56 @@ def _calc():
         unmount(conn, tmp_repo)
 
 
+def _execute_custom(conn, node, output):
+    command, args = parse_custom_command(node)
+
+    # Locate the command in the config file and instantiate it.
+    cmd_fq_class = CONFIG.get('commands', {}).get(command)
+    if not cmd_fq_class:
+        raise SplitGraphException("Custom command {0} not found in the config! Make sure you add an entry to your"
+                                  " config like so:\n  [commands]  \n{0}=path.to.command.Class".format(command))
+
+    ix = cmd_fq_class.rindex('.')
+    try:
+        cmd_class = getattr(import_module(cmd_fq_class[:ix]), cmd_fq_class[ix + 1:])
+    except AttributeError as e:
+        raise SplitGraphException("Error loading custom command {0}".format(command), e)
+    except ImportError as e:
+        raise SplitGraphException("Error loading custom command {0}".format(command), e)
+
+    with conn.cursor() as cur:
```

Comment:
This schema qualification doesn't get reset back to public which might be a source of some bugs in the future, though note that we always schema-qualify throughout the codebase and this is just for the life of the connection (which gets closed after we've run the sgfile)